projects/ngneat/error-tailor/src/lib/control-error.directive.ts

```diff
@@ -94,10 +98,21 @@ export class ControlErrorsDirective implements OnInit, OnDestroy {
     if (this.controlErrorsClass) {
       instance.customClass = this.controlErrorsClass;
     }
+
+    if (this.mergedConfig.controlErrorComponentAnchorFn) {
+      this.customAnchorDestroyFn = this.mergedConfig.controlErrorComponentAnchorFn(
+        this.host.nativeElement as HTMLElement,
+        (this.ref.hostView as EmbeddedViewRef<any>).rootNodes[0] as HTMLElement
+      );
+    }
   }
 
   ngOnDestroy() {
     this.destroy.next();
+    if (this.customAnchorDestroyFn) {
```

Comment:
I'm pretty sure you need to call it when there is no error, and not only on destroying.

Reply:
As far as I can see when there's no error, the error component is hidden using `hide-control` css class. It seems to be destroyed only when ControlErrorsDirective.ngDestroy() is called `this.ref = null`. Unless I'm missing something.

Reply:
Just make sure we are not recreating the element each time there is a change.

Reply:
I did verify that. Changes are ready, but need to review before I commit. A bit busy with work commitments. Will get to it as soon as I'm done with those.